gnovm/stdlibs/std/native.gno

```diff
@@ -10,8 +10,9 @@ func AssertOriginCall() // injected
 // MsgRun.
 func IsOriginCall() bool // injected
 
-func GetChainID() string // injected
-func GetHeight() int64   // injected
+func GetChainID() string     // injected
+func GetChainDomain() string // injected
```

Comment:
There was a discussion on `PrevRealm` that was about removing all the `Get` prefixes, is it's still accurate ?

In this case, `std.ChainID`, `std.ChainDomain` and `std.Height`

Reply:
Yep, we'll remove in a refactor, right now it makes sense to just be consistent.